tests/config/config.d/forbidden_get_client_http_headers.xml

```diff
@@ -0,0 +1,4 @@
+<clickhouse>
+    <get_client_http_header_forbidden_headers>['FORBIDDEN-KEY1','FORBIDDEN-KEY2','FORBIDDEN-KEY,CONTAINS_COMMA']</get_client_http_header_forbidden_headers>
```

Comment:
XML is a structured language, and array formatting is another structured language. We shouldn't mix both.